contracts/truefi2/PoolFactory.sol

```diff
@@ -295,4 +298,16 @@ contract PoolFactory is IPoolFactory, UpgradeableClaimable {
         safu = _safu;
         emit SafuChanged(_safu);
     }
+
+    /**
+     * @dev Calculate total TVL in USD
+     * @return _tvl TVL for all supported pools
+     */
+    function tvl() public override view returns (uint256) {
```

Comment:
```suggestion
    function supportedPoolsTVL() public override view returns (uint256) {
```

We should probably disambiguate which pools we're calculating TVL from.

Rationale: we have several other pool-related mappings to track in PoolFactory. It's complicated and I wish we could simplify this, but these have already been deployed.
```
    mapping(address => address) public pool;
    mapping(address => bool) public override isPool;
    mapping(address => bool) public isAllowed;
    mapping(address => mapping(address => address)) public singleBorrowerPool;
```

I'm worried we might forget in the future and think that `tvl()` is calculated from the `isPool() == true` set.